app/services/bulk_update/recommendations_uploads/validate_csv_row.rb

```diff
@@ -41,6 +41,17 @@ def validate!
         phase
         age_range
         subject
+        if csv_row_empty?(row)
+          messages = []
+        end
+      end
+
+      def csv_row_empty?(row)
+        row.instance_variables.each do |variable|
+          value = row.instance_variable_get(variable)
+          return false unless value.nil?
```

Comment:
Using `blank?` here also covers empty strings ("").
```suggestion
          return false unless value.blank?
```